test.js

```diff
@@ -706,3 +706,49 @@ test('calling cancel method on a process which has been killed does not make err
 	const {isCanceled} = await t.throwsAsync(subprocess);
 	t.false(isCanceled);
 });
+
+test.cb('fork()', t => {
+	const fork = execa.fork('fixtures/noop');
+
+	fork.on('close', code => {
+		t.is(code, 0);
+		t.end();
+	});
+});
+
+test('fork pipe stdout', async t => {
+	const {stdout} = await execa.fork('fixtures/noop', ['foo'], {
+		stdout: 'pipe'
+	});
+
+	t.is(stdout, 'foo');
+});
+
+test('fork correctly use execPath', async t => {
+	const result = await execa.fork(process.platform === 'win32' ? '/c hello.cmd' : 'hello.sh', [], {
```

Comment:
Please use destructuring: `const {stdout} = ...`

Same for the other test below.